CHANGELOG.rst

```diff
@@ -1,6 +1,109 @@
+- Add ``GetMetricHistory`` client API in Python and Java corresponding to the REST API. (#1178, @smurching)
+- [Tracking][R] API additions to be at parity with REST API and Python (#1122, @kevinykuo)
+- [Artifacts] Hadoop artifact repository with Kerberos authorization support  (#1011, @jaroslawk)
+- [Artifacts] Don't copy local artifacts in ``ArtifactRepository.download_artifacts`` to avoid having many copies of large model files in serving (#1307, @andrewmchen)
+- Support GCS in download utilities. ``gs://bucket/path`` files are now supported by the ``mlflow download`` CLI command and as parameters of type ``path`` in MLProject files. (#1168, @drewmcdonald)
+- [Python][Models] All python models exported by MLflow now declare ``mlflow`` as a dependency by default. In addition, we introduce a flag ``--install-mlflow`` users can pass to ``mlflow models serve`` and ``mlflow models predict`` methods to force installation of the latest version of MLflow into the model's environment. (#1308, @tomasatdatabricks)
+- [Python][Models] Update model flavors to lazily import dependencies. Modules that define Model flavors now import extra dependencies such as ``tensorflow``, ``scikit-learn``, and ``pytorch`` inside individual _methods_, ensuring that these modules can be imported and explored even if the dependencies have not been installed on a user's system. Also, the ``DEFAULT_CONDA_ENVIRONMENT`` module variable has been replaced with a ``get_default_conda_env()`` function for each flavor.
+- [Serving] Switch to ``gunicorn`` for serving of python models for better performance. This does not change the user interface. (#1322, @tomasatdatabricks)
+- [SageMaker][Deployment] Use the uniquely-generated model name as the S3 bucket prefix instead of requiring one from user. (#1183, @dbczumar)
+- [REST API] Add support for non-preview API paths. The ``preview`` paths will be deprecated in a future version of MLflow. (#1236, @mparkhe)
+
+Bug fixes and documentation updates:
+
+- Log metric timestamps in milliseconds by default (#1177, @smurching; #1333, @dbczumar)
+- Update artifact repository download methods to return absolute paths (#1179, @dbczumar)
+- Make FileStore respect the default artifact location (#1332, @dbczumar)
+- [DB] Fix bug when deserializing integer experiment ID for runs in SQLAlchemyStore (#1167, @smurching)
+- [DB] Ensure unique constraint names in MLflow tracking database (#1292, @smurching)
+- [R] Fix base64 encoding for basic auth in R tracking client (#1126, @freefrag)
+- [CLI] Correctly handle ``file:`` URIs for the ``-—backend-store-uri`` option in ``mlflow server`` and ``mlflow ui`` (#1171, @eedeleon, @tomasatdatabricks)
+- [Artifacts] Fix ``log_artifact`` failures due to existing directory on FTP server (#1327, @kafendt)
+- [Artifacts] Fix GCS artifact logging of subdirectories (#1285, @jason-huling)
+- [Projects] Fix bug not sharing sqlite db file with docker container (#1347, @tomasatdatabricks)
```

Comment:
Fix bug not sharing sqlite db file with docker container  -> Fix bug not sharing SQLite database file with Docker container